extension/package.json

```diff
@@ -5,7 +5,9 @@
   "prettier": "@stellar/prettier-config",
   "scripts": {
     "build": "webpack --config webpack.prod.js",
+    "build:experimental": "yarn build --env.EXPERIMENTAL=true",
     "start": "webpack-dev-server --config webpack.dev.js",
+    "start:experimental": "yarn start --env.EXPERIMENTAL=true",
```

Comment:
adding a flag called EXPERIMENTAL as this can be a long lived flag that lives beyond v2

Reply:
Hm maybe dev mode should always be experimental? I'm wary of stuff that adds in a decision point on what command to run to test changes. Production + experimental production makes sense, but dev mode seems like it should always have track the bleeding edge

Reply:
Ya I actually had it default to true in a previous version for the same reason, but my thought is if this is going to be an open source project and we want people to be able to jump in and mess around easily, the default behavior should be to mirror what people are seeing in the production version. That always helps me get my bearings in a new codebase